modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,95 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-0_{context}"]
+= Release notes for {gitops-title} 1.12.0
+
+{gitops-title} 1.12.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-12.0_{context}"]
+== Errata updates
+
+TBD
+
+[id="new-features-1-12-0_{context}"]
+== New features
+
+* With this update, the Argo CD CLI is supported and accessible as a productized and easily-installable RPM. The Argo CD CLI tool enables users to manage Red Hat OpenShift GitOps from a terminal. Using this tool, users can manage ArgoCD resources, such as Application, ApplicationSets, AppProjects, User accounts, and GPG keys from a client.
+
+[NOTE]
+====
+The `argocd` executable binary file is included in the archives and the RPMs.
+====
+
+[WARNING]
+==== 	
+The Argo CD CLI tool is a Technology Preview feature only.
+====
+
+* With this update, two new fields `.spec. applicationSet.sourceNamespaces` and `.spec. applicationSet.scmProviders` are introduced in ArgoCD CRD to support `ApplicationSet` in any namespaces. Administrators use it to define certain namespaces that manage ApplicationSet resources.
+
+* With this update, Argo CD server has the required permissions to manage `ApplicationSet` resources.
+
+* With this update, you can configure log levels such as "debug", "info", "warn", "error", "panic" and "fatal". The default log level set for the output is "info". Users can select an appropriate log level by adding the environment variable `LOG_LEVEL` in Red Hat OpenShift GitOps subscription's field, `spec.config.env`.
```

Comment:
Not 100% sure of docs standards wrt release notes, but I think the log levels should be in the format: `debug`, `info`, `warn`, `error`, `panic` and `fatal`. (backticks instead of quotes) 

If you look here under ApplicationSet logLevel they are like that. https://docs.openshift.com/gitops/1.11/argocd_instance/argo-cd-cr-component-properties.html#argo-cd-properties_argo-cd-cr-component-properties

Reply:
Fixed.